src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/TimeoutTests.cs

```diff
@@ -0,0 +1,201 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Security.Cryptography.X509Certificates.Tests.Common;
+using System.Linq;
+using Xunit;
+
+namespace System.Security.Cryptography.X509Certificates.Tests.RevocationTests
+{
+    [OuterLoop("These tests exercise timeout properties which take a lot of time.")]
+    public static class TimeoutTests
+    {
+        [Theory]
+        [InlineData(PkiOptions.OcspEverywhere)]
+        [InlineData(PkiOptions.CrlEverywhere)]
+        [PlatformSpecific(TestPlatforms.Windows | TestPlatforms.Linux)]
+        public static void RevocationCheckingDelayed(PkiOptions pkiOptions)
+        {
+            CertificateAuthority.BuildPrivatePki(
+                pkiOptions,
+                out RevocationResponder responder,
+                out CertificateAuthority rootAuthority,
+                out CertificateAuthority intermediateAuthority,
+                out X509Certificate2 endEntityCert,
+                nameof(RevocationCheckingDelayed));
```

Comment:
Isn't this parameter defaulted to [CallerMemberName]?

Reply:
Nope https://github.com/dotnet/runtime/blob/3642deea8235ce8316db8c37cc3e8f4ad5b15380/src/libraries/Common/tests/System/Security/Cryptography/X509Certificates/CertificateAuthority.cs#L821

Reply:
Ah, the DynamicRevocationTest wrapper is:

https://github.com/dotnet/runtime/blob/3642deea8235ce8316db8c37cc3e8f4ad5b15380/src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/DynamicRevocationTests.cs#L1273-L1281

You could local wrap here, or call that one.  Or just specify the names.